airflow/providers/apache/kafka/operators/consume_from_topic.py

```diff
@@ -0,0 +1,170 @@
+    :param apply_function: The function that should be applied to fetched one at a time.
+        name of dag file executing the function and the function name delimited by a `.`
+    :param apply_function_batch: The function that should be applied to a batch of messages fetched. Can not
+        be used with `apply_function`. Intended for transactional workloads where an expensive task might
+        be called before or after operations on the messages are taken.
+    :param apply_function_args: Additional arguments that should be applied to the callable, defaults to None
+    :param apply_function_kwargs: Additional key word arguments that should be applied to the callable
+        , defaults to None
+    :param consumer_config: the config dictionary for the kafka client (additional information available on the
+        confluent-python-kafka documentation), defaults to None, defaults to None
+    :param commit_cadence: When consumers should commit offsets ("never", "end_of_batch","end_of_operator"), defaults to "end_of_operator";
+         if end_of_operator, the commit() is called based on the max_messsages arg. Commits are made after the operator has processed the apply_function method for the maximum messages in the operator.
+         if end_of_batch, the commit() is called based on the max_batch_size arg. Commits are made after each batch has processed by the apply_function method for all messages in the batch.
+         if never,  close() is called without calling the commit() method.
+    :param max_messages: The maximum total number of messages an operator should read from Kafka,
+         defaults to None
+    :param max_batch_size: The maximum number of messages a consumer should read when polling, defaults to 1000
+    :param poll_timeout: How long the Kafka consumer should wait before determining no more messages are available,
+         defaults to 60
+    """
+
+    BLUE = "#ffefeb"
+    ui_color = BLUE
+    template_fields = (
+        "topics",
+        "apply_function",
+        "apply_function_args",
+        "apply_function_kwargs",
+        "config",
+    )
+
+    def __init__(
+        self,
+        topics: str | Sequence[str],
+        apply_function: Callable[..., Any] | str | None = None,
+        apply_function_batch: Callable[..., Any] | str | None = None,
+        apply_function_args: Sequence[Any] | None = None,
+        apply_function_kwargs: dict[Any, Any] | None = None,
+        consumer_config: dict[Any, Any] | None = None,
+        commit_cadence: str | None = "end_of_operator",
+        max_messages: int | None = None,
+        max_batch_size: int = 1000,
+        poll_timeout: float | None = 60,
+        **kwargs: Any,
+    ) -> None:
+
+        super().__init__(**kwargs)
+
+        self.topics = topics
+        self.apply_function = apply_function or None
+        self.apply_function_batch = apply_function_batch or None
```

Comment:
```suggestion
        self.apply_function = apply_function
        self.apply_function_batch = apply_function_batch
```
Since `None` is the default.